packages/tools/pie-icons/index.d.ts

```diff
@@ -0,0 +1,8 @@
+type Icon = {
+    name: string;
+    contents: string;
+    attrs: object;
+    pathPrefix: string;
+}
+
+declare const icons: Record<string, Icon>;
```

Comment:
🫶

Reply:
A record is a bit like a C# Dictionary, so this is made up of a series of KVPs where the key is the name of the icon and the value is an object representing the icon. The record can hold all of the icons so it doesn't need to be an array 👍🏻

Reply:
Oh, it was just a heart emoji 😂 thanks for the summary of Records tho!